lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
No, I don't think so, it just requires lib/agent.js, which requires adapter.js, which only requires appmetrics when 'start' is called.